woocommerce/woocommerce-ios
# [Shipping labels] Use backend data for carrier (predefined) packages

#14506


## Description


- Updated WooShippingAddPackageViewModel to enable cleaner carrier packages tab UI

## Testing information


- Install and set up the Woo Shipping extension on your store.
- Build and run the app with the revampedShippingLabelCreation feature flag enabled.
- Create an order with the processing status and at least one physical product.
- In the order details, select "Create Shipping Label."
- Store options will start fetching from backend in the background.
- Tap "Select a Package" button
- Switch to "Carrier" tab and check that the carrier/predefined packages are showing below in the lists

## Screenshots


https://github.com/user-attachments/assets/ce51f62c-7615-4fc0-822b-fc16c5a3bc91

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
